queries.sql

```diff
@@ -117,4 +117,52 @@ SELECT a.name, a.escape_attemps from animals a
 SELECT o.full_name, count(a.owner_id) as owns from owners o
     JOIN animals a ON o.id = a.owner_id
         GROUP by o.full_name
-            HAVING MAX(a.owner_id) = 3;
\ No newline at end of file
+            HAVING MAX(a.owner_id) = 3;
+
+/* ------------------------------------------------------------- */
+
+SELECT a.name from animals a 
+    JOIN visits v ON a.id = v.animal_id
+    JOIN vets ve ON v.vet_id = ve.id
+        WHERE v.date_of_visit = (SELECT MAX(date_of_visit) from visits 
+                                    WHERE vet_id = (SELECT id from vets 
+                                                        WHERE name = 'William Tatcher'));       
```

Comment:
- [ ] What the previous reviewer meant was that you add your screen on the pull request as I have done below 👇🏾  Instead of a link to a bunch of unnamed screenshots. Please add this and label each image as I have done below.

- The query you did. example - Who was the last animal seen by William Tatcher?
![image](https://user-images.githubusercontent.com/84156714/216629425-4a12e4a3-28c1-49b2-a363-b657a7fa810e.png)

- The query you did. example - How many different animals did Stephanie Mendez see?
![image](https://user-images.githubusercontent.com/84156714/216629425-4a12e4a3-28c1-49b2-a363-b657a7fa810e.png)

This way it is easy to see the result of your query if they are correct. Currently, the screenshots are not named or labeled, thus, one cannot tell which query is the screenshot for.

Reply:
Hello @bobb-Rob. Is that the only change I need to make?
I need to be sure, it's my last available review.

Reply:
I cannot tell if this is the only change required because I could not understand your screenshots query results, hence could not make any comments regarding the correctness of the queries. But from what I saw in your query file, it looks like your code was ok. Just ensure your pull request shows all screenshots with labels. Then the next reviewer will make comments or approve that.

Reply:
Hello @enoqueJonas Sorry for the late reply but I see that @bobb-Rob helped you a lot, thanks @bobb-Rob for your thorough explanation 🙏🏻